with-firebase-saml-login/App.js

```diff
@@ -0,0 +1,126 @@
+// Import third-party dependencies
+import React from 'react'
+import { View, Button, LogBox, StyleSheet, Text } from 'react-native'
+import * as WebBrowser from 'expo-web-browser';
+import * as Linking from 'expo-linking';
+
+import firebase from 'firebase'
+
+const firebaseConfig = {// !!!!!PUT YOUR FIREBASE CONFIG HERE!!!!!
+  apiKey: "AIzaSyAlZruO2T_JNOWn4ysfX6AryR6Dzm_VVaA",
+  authDomain: "blobtest-36ff6.firebaseapp.com"
+};
+
+if (!firebase.apps.length) {
+  firebase.initializeApp(firebaseConfig);
+}
+
+// Firebase sets some timeers for a long period, which will trigger some warnings. Let's turn that off for this example
+LogBox.ignoreLogs([`Setting a timer for a long period`]);
+/**
+ * @author Tag Howard
```

Comment:
We utilize a shared contributors section for this type of attribution https://github.com/expo/examples#contributors-

Reply:
My bad, used to copy/pasting my usual @author @version on every function I write